test/parallel/test-http-agent-keepalive.js

```diff
@@ -52,7 +52,7 @@ function get(path, callback) {
     port: server.address().port,
     agent: agent,
     path: path
-  }, callback);
+  }, callback).on('socket', common.mustCall(checkListeners));
```

Comment:
Oops, probably needs to actually be this?

```suggestion
  }, callback).on('socket', common.mustCall(checkListeners, 4));
```

Reply:
Oh, no, that's wrong, i had it right the first time. Will delete above and this comment.